deploy/aws/README.md

```diff
@@ -76,7 +76,7 @@ monitor_endpoint = http://abd299cc47af411e98aae02938da0762-1989524000.us-east-2.
 region = us-east-2
 tidb_dns = abd2e3f7c7af411e98aae02938da0762-17499b76b312be02.elb.us-east-2.amazonaws.com
 tidb_port = 4000
-tidb_version = v3.0.0-rc.1
+tidb_version = v2.1.8
```

Comment:
Oh, I forgot to upgrade the tidb versions for aws and gcp deployment. I'll send a PR later to update the versions.

Reply:
I've sent a PR to fix this #532